### DISCUSSION TITLE
What is the purpose of Basic-Auth in the gatsby-plugin-cloud documentation?

### DISCUSSION THREAD

**Discussion by njbmartin** [gatsbyjs/gatsby#36066] [Help]
I'm looking at implementing extremely rudamentary authentication using Basic Auth, and the cloud documentation seems to suggest that Basic Auth can be enabled on Gatsby Cloud through the headers:

```js
{
  options: {
    headers: {
      "/*": [
        "Basic-Auth: someuser:somepassword anotheruser:anotherpassword",
      ],
      "/my-page": [
        // matching headers (by type) are replaced by Gatsby Cloud with more specific routes
        "Basic-Auth: differentuser:differentpassword",
      ],
    },
  }
}
```

However, this seems to have no effect. The expectation was that it would prompt the user to login in order to access it, but it does nothing other than reveal it in the headers.

**Comment by graysonhicks** [comment]
Hey Nico -

The point of those headers is not to _create_ basic auth, but to pass those headers to other parts of your infrastructure that may already be password protected (your CMS). You can currently password protect gtsb.io domains in the site settings tab.  We are rolling our password protection for gatsbyjs.io production domains soon!
